internal/cli/roles_permissions.go

```diff
@@ -0,0 +1,254 @@
+package cli
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/spf13/cobra"
+	"gopkg.in/auth0.v5/management"
+)
+
+var (
+	roleAPIIdentifier = Flag{
+		Name:       "API",
+		LongForm:   "api-id",
```

Comment:
This should just probably be an argument, like the other subresources in the CLI:
- `auth0 users blocks list <user-id>`
- `auth0 apis scopes list <id|audience>`

Reply:
The difference here is we need two separate things:

1. The role ID
2. On top of that, the api-id.

Do you mean to say two arguments?

Reply:
Oh, you're right 🤦🏼‍♀️